src/content/docs/apis/nerdgraph/examples/nerdgraph-slm.mdx

```diff
@@ -0,0 +1,136 @@
+
+The following is an example NerdGraph call that creates an SLI using the `serviceLevelCreate` mutation query:
+
+```
+mutation { 
+  serviceLevelCreate(
+    entityGuid: "entityGuid", 
+    indicator: {
+      name: "Latency below 0.25 seconds",
+      description: "The proportion of valid requests that were served faster than 0.25s, which is considered to correspond to a good experience.", 
+      events: {
+        validEvents: {
+          from: "Transaction", 
+          where: "entityGuid = 'entityGuid'"
+        }, 
+        goodEvents: {
+          from: "Transaction", 
+          where: "entityGuid = 'entityGuid' and duration < 0.25"
+        }, 
+        accountId: accountId}, 
+        objectives: {
+          target: 99.5, 
+          timeWindow: {
+            rolling: {
+              count: 7, 
+              unit: DAY
+       }}}}) 
+  {
+    id
+    description
+  }
+}
+```
+
+It contains these fields:
+
+* **`entityGuid`**: The GUID of the [entity](https://docs-preview.newrelic.com/docs/service-level-management/create-slm#sli-entity) (for example, APM service, browser application, etc.) that you want to relate this SLI to. On the UI, you can find this GUID on the entity page, under **See metadata and manage tags**.
+* **`description`**: Use detailed descriptions, including the selected threshold that determines good events. 
+  * For example, for an availability SLI, include something like “The proportion of valid requests that were served without errors.” 
+  * Or, for a latency SLI, include a description such as “The proportion of valid requests that were served faster than 0.25s, which is considered to correspond to a good experience”. 
+* **`accountId`**: The ID of the account where the APM service or browser application belongs to, which contains the NRDB data for the SLI/SLO calculations. 
+* **`badEvents.from`**, **`badEvents.where`** 
+  * The NRQL query that defines bad events, `SELECT count(*) FROM badEvents.from WHERE badEvents.where`, requires these `FROM` and `WHERE` clauses.  
+  * If you defined an SLI from valid and bad events, leave the `goodEvents` object empty. 
+* **`goodEvents.from`**, **`goodEvents.where`** 
+  * The NRQL query that defines good events, `SELECT count(*) FROM goodEvents.from WHERE goodEvents.where`, requires these `FROM` and `WHERE` clauses.  
+  * If you defined an SLI from valid and good events, leave the `badEvents` object empty. 
+* **`validEvents.from`**, **`validEvents.where`**
+  * These are the `FROM` and `WHERE` clauses for the NRQL query that defines valid events, which will result in `SELECT (*) FROM validEvents.from WHERE validEvents.where`. 
```

Comment:
Hi @urbiz-nr ! The query function is count(*) here as well, please. Apologies for introducing this bug on the original draft.

Reply:
No problem, I'm committing the fix in a sec!